File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_LYDONGHIPHEPService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_LYDONGHIPHEPService : BaseService<NS_DM_LYDONGHIPHEP, INS_DM_LYDONGHIPHEPRepository>
    , INS_DM_LYDONGHIPHEPService
    {
    private readonly INS_DM_LYDONGHIPHEPRepository _NS_DM_LYDONGHIPHEPRepository;
    public NS_DM_LYDONGHIPHEPService(INS_DM_LYDONGHIPHEPRepository NS_DM_LYDONGHIPHEPRepository) : base(NS_DM_LYDONGHIPHEPRepository)
    {
    _NS_DM_LYDONGHIPHEPRepository = NS_DM_LYDONGHIPHEPRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_LYDONGHIPHEP.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_LYDONGHIPHEP } from './NS_DM_LYDONGHIPHEP.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_LYDONGHIPHEPService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaLyDoNghiPhep', 'TenLyDoNghiPhep'];

    List: NS_DM_LYDONGHIPHEP[] | undefined;
    ListFilter: NS_DM_LYDONGHIPHEP[] | undefined;
    FormData!: NS_DM_LYDONGHIPHEP;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_LYDONGHIPHEP";
    }
}

